proto/stat.proto

```diff
@@ -46,16 +46,25 @@ message PointStat {
   // date in IOS-8601 format
   string date = 1;
   double value = 2;
+  // Full metadata of the stat
   StatMetadata metadata = 3;
+  // A hash string of the meta data, this is to be used together with a map from
+  // the hash to the full metatdata.
+  string meta_hash = 4;
 }
 
 message PlacePointStat {
   // Keyed by place DCID.
   map<string, PointStat> stat = 1;
-  // Keyed by import name.
+  // Keyed by metadata hash.
```

Comment:
I wonder why not make it an index into metadata_list below? (fewer bytes stored overall)

Reply:
Good point. Moved the map as response level information.